flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -94,6 +94,21 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "his is a test String.")
   }
 
+  @Test
+  def testReplace(): Unit = {
```

Comment:
add more test for invalid input such : empty string, null or a string is not available of the original string